plugins/godot-gradle-plugin/build.gradle.kts

```diff
@@ -31,4 +31,28 @@ dependencies {
     implementation(kotlin("stdlib"))
     implementation(kotlin("gradle-plugin"))
     implementation(kotlin("gradle-plugin-api"))
+}
+
+tasks {
+    val sourceJar by creating(Jar::class) {
+        archiveBaseName.set(project.name)
+        archiveVersion.set(project.version.toString())
+        archiveClassifier.set("sources")
+        from(sourceSets["main"].allSource)
+    }
+}
+
+publishing {
+    publications {
+        // this is only used for publishing locally.
+        val godotPlugin by creating(MavenPublication::class) {
+            pom {
+                groupId = "com.utopia-rise"
```

Comment:
Sorry could have seen this earlier...
Can't we define this globally somewhere? as it's the same for all publications in all modules?

Reply:
I'll use `project.group` for now as it has the value set to `com.utopia-rise`. We can generalise this later once we have the other modules that needs publishing.